### ISSUE TITLE
Deeplearning test case clarification

### ISSUE THREAD

**Issue by hfp** [hfp/libxsmm#359]
The following forward convolution misses accuracy with any build setting, compiler, or code path:

```bash
CHECK=1 ./layer_example_f32  1   27  27  8   128  128   3 3 1 1 1 F T 1
```

This is likely caused by an implementation defect rather than an accuracy issue. Notably, the kernel is padded.

If the issue is expected (padding not supported as per code-version), all test cases and scripts need to be updated accordingly.

**Comment by alheinecke** [comment]
This should be fixed with 29e6798209534f3be10c6adefab2a44f670ed11d

**Comment by hfp** [comment]
Fixed.
